src/SFA.DAS.EmployerAccounts.UnitTests/Commands/RemoveLegalEntityTests/WhenIValidateTheCommand.cs

```diff
@@ -96,86 +88,6 @@ public async Task ThenTheUserIsCheckedToSeeIfTheyAreAnOwnerOnTheAccount()
             Assert.IsTrue(actual.IsUnauthorized);
         }
 
-        [Test]
-        public async Task ThenTheAgreementIsCheckedToSeeIfItHasBeenSignedAndHasActiveCommitments()
```

Comment:
Lost a chunk of tests here, know some are due to some of the validators being changed and not throwing unauthorized in places but are these covered elsewhere?

Reply:
The tests have been moved, along with the logic, to the command handler.